src/main/java/io/jenkins/plugins/extratoolinstallers/installers/AuthenticatedDownloadCallable.java

```diff
@@ -54,45 +54,50 @@ class AuthenticatedDownloadCallable extends MasterToSlaveFileCallable<Date> {
     @CheckForNull
     private final TaskListener logOrNull;
 
+    private final boolean fallbackToExistingInstallation;
+
     /**
      * Passed to {@link FilePath#act(hudson.FilePath.FileCallable)} in order to
      * run
      * {@link #doDownload(HttpRequestBase, FilePath, TaskListener, URI, String, String)}
      * on a remote node.
-     * 
+     *
      * @param uri
      *            What to download.
      * @param usernameOrNull
      *            Username to authenticate as, or null for an anonymous
-     *            download.
+     *                                       download.

```

Comment:
Please do not changes such empty spaces. It makes it more difficult to review

Reply:
Was automatically done. I'll update the PR in a few minutes